packages/opds-web-client/src/components/Root.tsx

```diff
@@ -258,6 +276,17 @@ export class Root extends React.Component<RootProps, any> {
               />
           }
         </div>
+        { showFooter &&
+          <footer style={ footerStyle }>
+            <ul aria-label="about links" className="list-inline">
+              { this.props.collectionData.aboutLinks.map(link =>
+                <li key={link.url} style={{padding: "5px"}}>
+                  <a href={link.url}>{link.text}</a>
+                </li>
+              ) }
+            </ul>
+          </footer>
+        }
```

Comment:
I might need to make it possible to pass in a Footer component the way we did with Header. I'm not sure if it makes sense to show these links in the admin interface, and if not, the admin interface could pass in an empty Footer component.